ecs-logs-collector.sh

```diff
@@ -343,6 +341,11 @@ get_docker_logs() {
     amazon)
       cp /var/log/docker ${dstdir}
       ;;
+    amazon2)
+      if [ -e /bin/journalctl ]; then
+        /bin/journalctl -u docker > ${dstdir}/docker
```

Comment:
Let's quote the use of the `${dstdir}` here for expansion's sake and not call `journalctl` by absolute path, instead lets let it be anywhere on the path with plain ol `journalctl -u docker`.
```suggestion
        journalctl -u docker > "${dstdir}"/docker
```

Reply:
That sounds good for me. I just kept the same format as other Systemd systems, so if we change this we should change the others too. Also, every mention to `${dstdir}` is not quoted, I would leave these changes for the next update so they all are updated in the same commit.

Reply:
I'm all for incremental improvements, and though the precedent exists (much of this appears to be copied from there), I would prefer we take this opportunity to make this addition better than what came before :)